src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -54,10 +55,13 @@ public abstract class PlatformInstallerBase
                 .AppendLine($"cd {versionDirInTemp}")
                 .AppendLine("PLATFORM_BINARY_DOWNLOAD_START=$SECONDS")
                 .AppendLine($"platformName=\"{platformName}\"")
+                .AppendLine($"if [ \"$DEBIAN_FLAVOR\" = \"bullseye\" ]; then")
```

Comment:
Could you please clarify how this only applies to PHP 7.4?
Also, this would definitely deserve a comment in the codebase to clarify why we're doing that, and the link to the work item to revert it, etc. Could you please add this?

Reply:
Hi Dan, I believe the issue not only happens for PHP 7.4, but also other PHP and PHP composer versions. As we know of, we added an if else statement for PHP and PHP composer for an old ICM issue, so it will select SDKs according to its image debian type. But now we have bullseye image, so it'll apply to bullseye SDKs for PHP and PHP composer, which will definitely fail. So we need to select buster SDKs until bullseye SDKs are supported.

Yes sure, I can add comment and link above it.